subscriptions/templatetags/seed.py

```diff
@@ -0,0 +1,10 @@
+from django import template
+
+import seed_stage_based_messaging
+
+register = template.Library()
+
+
+@register.simple_tag
+def current_version():
+    return seed_stage_based_messaging.__version__
```

Comment:
The other way of doing that is using [pkg_resources](https://github.com/praekelt/mc2/blob/develop/mc2/context_processors.py#L15-L19)